### ISSUE TITLE
[Remoto] Especialista em Inteligência Artificial @ ITeam

### ISSUE THREAD

**Issue by ITeam-oportunidades** [programadores-br/geral#2141]
> Vaga Remota 

## ITeam

Há mais de 15 anos presente no mercado, a ITeam oferece o comprometimento com o cliente.
Reunimos capital intelectual especializado por profissionais experientes e capacitados a prestar os melhores serviços nas diferentes áreas de atuação, submetidos a treinamentos constantes para estarem sempre atualizados com as melhores práticas de mercado. 
Com uma equipe multidisciplinar, a ITeam proporciona uma área de Serviços Gerenciáveis como Projetos SAP, Fábrica de Software, Fábrica de Soluções e Sustentação. Neste modelo proporcionamos Sustentação, Melhorias e Desenvolvimentos de aplicação de forma integrada.

## Local

100% Remoto - Full time horário comercial

## Requisitos

Matemática: álgebra Linear, cálculo básico;
Estatística: Probabilidade, Distribuições e Testes de Hipóteses;
Linguagens: Python, Java;
Principais famílias de algoritmos de Machine Learning e Data Science;
Experiência com métricas de avaliação de modelos;
Data Engineering: data analysis, data mining, feature engineering;
Deep Learning: CNN, RNN e LSTM;
 Frameworks: TensorFlow, scikit-learn, PyTorch, Keras;
Plataformas: Jupyter Notebook, Anaconda, Assistentes Virtuais;

## Necessário

Inglês avançado para conversação

## Atividades

Desenvolver projetos de machine learning e Data Science;
Trazer a visão de IA para áreas de inovação, tecnologia e negócio;
Disseminar o conhecimento em machine learning;
Apoiar na disseminação da IA na empresa;
Liderança técnica de times de tecnologias;
Acompanhamento e validação técnica de experimentos de tecnologias de IA envolvendo fornecedores e startups;
Pesquisar e desenvolver soluções inovadoras utilizando técnicas e algoritmos de machine learning e deep learning;
Identificar oportunidades de exploração dos dados existentes na empresa para utilização de IA para geração de valor (tanto internamente para empresa, quanto para o cliente final).
Realizar atividades técnicas experimentais de IA, Data Science e Machine Learning.
Avaliar os fornecedores, soluções, frameworks, plataformas e arquiteturas para operacionalização de soluções que utilizam IA.

## Contratação

PJ ou a combinar
Tempo indeterminado

## Como se candidatar

Por favor, envie um e-mail para viviane.pereira@iteam.inf.br com seu CV anexado e pretensão de taxa/hora - enviar no assunto: Especialista IA

## Labels
Remoto
Inglês 
Python
Java
Machine Learning
Data Science
Matemática
Estatística

## Regime
PJ ou a combinar

## Nível
Sênior
